docs/machineset-azure.md

```diff
@@ -0,0 +1,70 @@
+# Creating an Azure Windows MachineSet
+
+_\<infrastructureID\>_ should be replaced with the output of:
+```shell script
+ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+```
+
+<location> should be replaced with a valid Azure location like centralus.
```

Comment:
Please follow the same formatting in [machineset-aws.md](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md) through this doc